mariadb.html.md.erb

```diff
@@ -47,4 +49,23 @@ Cloud Foundry: [Spring Music Example](https://github.com/cloudfoundry-samples/sp
 
 ## <a id='limitations'></a> Known Limitations
 
-The base of our HA MariaDB offering is built by a galera cluster, which brings some [known limitations](https://mariadb.com/kb/en/mariadb/mariadb-galera-cluster-known-limitations/) with it.
+Our HA MariaDB offering is based on Galera cluster, which brings some [known limitations](https://mariadb.com/kb/en/mariadb/mariadb-galera-cluster-known-limitations/) with it.
+
+Our Galera aaS is a shared Galera cluster. Each customer has their own database (`CREATE DATABSE`) with all privileges.
+
+```
+GRANT ALL PRIVILEGES ON `CF_DATABASE_NAME`.* TO 'USER'@'SUBNET'
+```
+
+## <a id='definer'></a> Definer and bind-service (Credentials changing)
+
+With `cf bind-service` the service broker (Cloud Foundry component) creates new credentials for security reasons (`cf env APP_NAME` to view them). This will break existing routines, triggers, views and events because they are still using the old user.
+
+Error Code from MariaDB backend:
+
+```
+Error: 1449 SQLSTATE: HY000 (ER_NO_SUCH_USER)
+Message: The user specified as a definer ('%s'@'%s') does not exist
+```
+
+See MariaDB Blog post [Changing the DEFINER of MySQL stored routines etc.](https://mariadb.com/resources/blog/changing-definer-mysql-stored-routines-etc) for ideas how to fix this.
```

Comment:
* "Blog" should be "blog"
* "for ideas how to..." should be "for ideas on how to..."